lib/pf/mfa/Akamai.pm

```diff
@@ -441,7 +441,7 @@ sub verify_response {
     my $response = decode_json($token->{payload});
     if ($response->{response}->{result} eq "ALLOW" && $response->{response}->{username} eq $username) {
         #MFA verification success
-        cache->set($username."mfapostauth", $TRUE, 5);
+        cache->set($username."mfapostauth", $TRUE, normalize_time($self->post_mfa_validation_cache_duration));
```

Comment:
I'm going to be mr. picky but this is copy/pasted between the 2 MFA modules, please centralize the logic of success/failure MFA calls in the cache to the base MFA module